search-service/src/main/kotlin/com/egm/stellio/search/service/EntityEventListenerService.kt

```diff
@@ -128,11 +132,18 @@ class EntityEventListenerService(
         temporalEntityAttributeService.getForEntityAndAttribute(
             entityId, expandedAttributeName, datasetId
         ).zipWhen {
+            val instanceId = "urn:ngsi-ld:Instance:${UUID.randomUUID()}".toUri()
```

Comment:
`instanceId` has already a default value (same as here) when you create an instance of `AttributeInstance`?